ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -398,6 +398,44 @@ To retain the previous behaviour, set the value to `false`:
 +shutdown_interfaces_towards_undeployed_peers: false
 ```
 
+### Default MTU value changed from `9214` to `9194` for Dps1, WAN HA and WAN p2p uplink interfaces
+
+In AVD 4.0.0, the default MTU for Dps1, WAN HA and WAN p2p uplink interfaces was `9214`.
+In AVD 5.0.0, the default MTU for Dps1, WAN HA and WAN p2p uplink interfaces has been changed to `9194`.
+
+To retain the old MTU on WAN HA interfaces use `<node_type>.defaults/node_groups[]/node_groups[].nodes[]/nodes[].wan_ha.mtu` as `9214`.
+
+```diff
+wan_router:
+  node_groups:
+    - group: SITE_HA_ENABLED
+      wan_ha:
+        enabled: true
+        ha_interfaces: [Ethernet52]
++       mtu: 9214
+```
+
+To retain the old MTU on Dps1 interface use custom structured configuration
+
+```diff
+custom_structured_configuration_dps_interfaces:
+  - name: Dps1
++   mtu: 9214
+```
+
+To retain the old MTU on WAN p2p uplink interfaces use custom_platform_settings
+
+```diff
+custom_platform_settings:
+  platforms:
+    - AWE-5310
+    - AWE-5510
+    - AWE-7250R
+    - AWE-7230R
+    - AWE-7220R
++ p2p_uplinks_mtu: 9214
```

Comment:
This example needs to include the other keys that are part of the defaults for these platforms, since this will override the existing platform settings (not merge).